packages/block-library/src/image/image-size.native.js

```diff
@@ -76,6 +77,14 @@ class ImageSize extends Component {
 			imageWidthWithinContainer: this.state.width,
 			imageHeightWithinContainer: this.state.height,
 		};
+		if ( this.state.width === undefined ) {
+			const width = 300;
+			return (
+				<View onLayout={ this.onLayout } style={ { flex: 1, flexDirection: 'row', justifyContent: 'center' } }>
+					<Dashicon icon={ 'format-image' } size={ width } />
+				</View>
+			);
+		}
```

Comment:
Returning early is fine but can be tricky understanding the code when skimming through. I think the code outside is rather trivial and small and I'd suggest we put it in an `else` block to help with readability. WDYT?

Reply:
> The lint rules in GB complain about that :).

Ah, there's `no-else-return` in place, right!